app/src/main/res/values/strings.xml

```diff
@@ -29,4 +29,18 @@
     <string name="coming_soon_text">Coming Soon</string>
     <string name="onSpeech">Turn On</string>
     <string name="offSpeech">Turn Off</string>
+    <string name="first_name">First Name</string>
+    <string name="last_name">Last Name</string>
+    <string name="phone_number">Phone Number</string>
+    <string name="btnsave">Save</string>
+    <string name="add_new_contacts">Add New Contacts</string>
+    <string name="btn_new">New</string>
+    <string name="btn_delete">Delete</string>
+    <string name="load">Load</string>
+    <string name="btnadd">Add</string>
+    <string name="textUserview">TextView</string>
+    <string name="textPhoneview">TextView</string>
+    <string name="back">Back</string>
+    <string name="phone_message">TODO</string>
+    <string name="phone_call">phone_call</string>
```

Comment:
Is there a way that we can use some of the older names and ids so that we don't have duplicates? Just take another look.

Reply:
I'll delete some of the string that doesn't required my MVP